contracts/MechMarketplace.sol

```diff
@@ -150,47 +150,6 @@ contract MechMarketplace is IErrorsMarketplace {
         );
     }
 
-    /// @dev Calculates payment and fee based on delivery rates and mech type.
-    /// @param mech Delivery mech address.
-    /// @param requestId Request Id.
-    /// @param deliveryRate Delivery rate.
-    /// @param mechPayment Mech payment.
-    /// @param marketplaceFee Marketplace fee.
-    function _calculatePayment(
```

Comment:
This is migrated to each balance tracker accordingly, and modified as per above.